### PR TITLE
[WIP] Add formats pie chart

### PR DESCRIPTION
**Pull request by ablwr**
Still needs cleanup / consideration, but proof-of-concept demo for #18

### CODE REVIEW

File: METSFlask/templates/aip.html [hunk 1]
```diff
@@ -16,24 +16,28 @@ <h2 class="h3">Descriptive metadata</h2>
...
-        <th>Format</th>
-        <th>Version</th>
-        <th>PUID</th>
-        <th>Size (bytes)</th>
-        <th>Last modified</th>
-        <th>UUID</th>
-        <th>Details</th>
-      </tr>
-    </thead>
-    <tbody>
```
Comment (tw4l) [inline METSFlask/templates/aip.html:35]: This bit appears to be getting overwritten with older code in this PR

**Comment by tw4l** [comment]
Hi Ashley! This is really great, thank you!

I've rebased the PR against a new branch where we can continue to work before merging into master.

Two notes:
* I think your changes might be based on an earlier commit than the current master, as some of the changes that have been made recently (e.g. improving the table semantics, using h2s with h3 classes for presentation) seem reverted here
* It would great to add ARIA roles and labels to the chart. For example:

```
<h2 class="h3">Data visualization</h2>
<div class="well" style="margin: 10px;">
  <canvas id="chart" width="400" height="400" aria-label="Format pie chart" role="img">
    <!-- Fallback text -->
    <p>Canvas element not supported.</p>
  </canvas>
</div>
```

**Comment by tw4l** [comment]
Btw, I really like the random color generation! It's a nice touch 😀

**Comment by tw4l** [comment]
I've invited you to be a collaborator on the repo. If it sounds good to you, I'll merge this into `timothyryanwalsh:dev/add-format-chart` and give you push rights so we can continue to work on this. I think this feature will be especially cool in combination with presenting all/other types of files besides 'original' to the user, which I'm planning to do soon.

**Comment by ablwr** [comment]
Okay great, I'll close this PR and work from the master dev/add-format-chart branch!

**Pull request merged**